pretrain/tinyllama.py

```diff
@@ -332,6 +329,16 @@ def init_weights(module: nn.Module, n_layer: int, n_embd: int):
             nn.init.normal_(param, mean=0.0, std=(1 / math.sqrt(n_embd) / n_layer))
 
 
+def choose_logger(logger_name: str, name: str, resume: Union[bool, Path], *args, **kwargs):
+    if logger_name == "csv":
+        return CSVLogger(root_dir="logs", name=name, *args, **kwargs)
+    if logger_name == "tensorboard":
+        return TensorBoardLogger(root_dir="logs", name=name, *args, **kwargs)
+    if logger_name == "wandb":
+        return WandbLogger(project="tinyllama", name=name, resume=(resume is not False), *args, **kwargs)
+    raise ValueError(f"`logger={logger} is not a valid option.")
```

Comment:
```suggestion
    raise ValueError(f"`logger={logger}` is not a valid option.")
```
I always late to the party 😞

Reply:
Feel free to drop this in any of your PRs or I'll do it in my next one.